roles/bootstrap-os/defaults/main.yml

```diff
@@ -12,3 +12,11 @@ coreos_locksmithd_disable: false
 ## General
 # Set the hostname to inventory_hostname
 override_system_hostname: true
+
+## Downloads
+# Pypy version
+coreos_pypy_version: "7.0.0"
```

Comment:
Maybe this should be `pypy3.5-7.0.0`? PyPy is a bit weird with its version numbers.